### PR TITLE
ES-2402: Use generated models in rest client

### PR DESCRIPTION
**Pull request by anton-subbotin**
### [ES-2402](https://r3-cev.atlassian.net/browse/ES-2402)

- Remove all importMappings to the existing and custom DTOs and use only generated models instead
- Remove `jsr310` dependency - generated models do not deal with `Instant` type
- Remove Gradle dependencies on inner projects

[ES-2402]: https://r3-cev.atlassian.net/browse/ES-2402?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by anton-subbotin** [comment]
build e2e

**Comment by anton-subbotin** [comment]
E2E tests using `5.3.0.0-alpha-1719248309321` version of Corda CLI: https://ci02.dev.r3.com/blue/organizations/jenkins/Corda5%2Fcorda5-e2e-tests/detail/PR-662/3/pipeline/
(PR: https://github.com/corda/corda-e2e-tests/pull/662/files)

**Comment by anton-subbotin** [comment]
E2E tests using `5.3.0.0-alpha-1719306441163` version of Corda CLI (removed `jsr310` dependency): https://ci02.dev.r3.com/blue/organizations/jenkins/Corda5%2Fcorda5-e2e-tests/detail/PR-662/4/pipeline/
(PR: https://github.com/corda/corda-e2e-tests/pull/662/files)

**Review by tlawson3 — DISMISSED** [review]

**Review by YashNabar — DISMISSED** [review]
LGTM (network-team files)

**Review by williamvigorr3 — DISMISSED** [review]
LGTM - Network team files

File: libs/rest/generated-rest-client/build.gradle [hunk 1]
```diff
@@ -96,7 +32,6 @@ dependencies {
     implementation libs.kotlin.stdlib
     implementation libs.jackson.dataformat.yaml
     implementation libs.jackson.databind
-    implementation libs.jackson.datatype.jsr310
     implementation libs.jackson.module.kotlin
     implementation libs.okHttp
     implementation project(':components:crypto:crypto-rest')
```
Comment (vkolomeyko) [inline libs/rest/generated-rest-client/build.gradle]: I think we ought to challenge all those dependencies below and potentially delete some/all of them.
Ideally, generated client should not depend on anything `project(.*rest.*)` and be produced just from `swaggerBaseline`.

File: libs/rest/generated-rest-client/build.gradle [hunk 2]
```diff
@@ -34,18 +34,6 @@ dependencies {
...
-    implementation project(':components:membership:membership-rest')
-    implementation project(':libs:configuration:configuration-endpoints')
-    implementation project(':libs:external-messaging')
-    implementation project(':libs:permissions:permission-endpoint')
-    implementation project(':libs:rest:rest')
-    implementation project(':libs:virtual-node:virtual-node-endpoints')
-    implementation project(':libs:virtual-node:virtual-node-info')
-    implementation "net.corda:corda-avro-schema:$cordaApiVersion"
-
-    testImplementation libs.javalin
```
Comment (tlawson3) [inline libs/rest/generated-rest-client/build.gradle:113]: I'm surprised we don't need the Javalin dependency for the tests

**Review by tlawson3 — APPROVED** [review]

**Review by vkolomeyko — APPROVED** [review]
LGTM

**Pull request merged**